server/block/block.go

```diff
@@ -216,13 +216,15 @@ func newFlammabilityInfo(encouragement, flammability int, lavaFlammable bool) Fl
 // EntityCollider is an interface for blocks with special behaviors on entity collision.
 type EntityCollider interface {
 	// EntityCollide is called on entity collision.
-	EntityCollide(e world.Entity)
+	EntityCollide(e world.Entity, pos cube.Pos)
```

Comment:
Maybe put `pos` before `e` for consistency reasons? Other block functions typically have `pos` as the first argument.

Reply:
I agree. I'll fix that tomorrow.

Reply:
Actually, I'll fix it now,